src/app/app.module.ts

```diff
@@ -25,12 +31,14 @@ import { Api } from '../providers/api';
 import { Settings } from '../providers/settings';
 import { Items } from '../mocks/providers/items';
 
-import { TranslateModule, TranslateLoader, TranslateStaticLoader } from 'ng2-translate/ng2-translate';
+import { TranslateModule, TranslateLoader } from '@ngx-translate/core';
+import { TranslateHttpLoader } from '@ngx-translate/http-loader';
+//TranslateStaticLoader
 
 // The translate loader needs to know where to load i18n files
 // in Ionic's static asset pipeline.
 export function createTranslateLoader(http: Http) {
-  return new TranslateStaticLoader(http, './assets/i18n', '.json');
+  return new TranslateHttpLoader(http, './assets/i18n', '.json');
```

Comment:
I did a search and there is no reference to TranslateStaticLoader on the ngx-translate page.
There are some references to "static".
This could be a result of the 5.0.0 to 6.0.1 transition with Angular 4.0.0 but I have not looked into it in depth. You could follow up with the author if needed.

AoT

If you want to configure a custom TranslateLoader while using AoT compilation or Ionic 2, you must use an exported function instead of an inline function.

```
export function createTranslateLoader(http: Http) {
    return new TranslateHttpLoader(http, './assets/i18n/', '.json');
}
```